lib/ash/resource/change/cascade_update.ex

```diff
@@ -210,7 +210,11 @@ defmodule Ash.Resource.Change.CascadeUpdate do
         return_notifications?: opts.return_notifications?
       )
 
-    context = Map.merge(relationship.context || %{}, %{cascade_destroy: true})
+    context =
+      Map.merge(relationship.context || %{}, %{
+        cascade_update: true,
```

Comment:
I `grep`ed through the project but couldn't find a place that looked for this and the tests are still working. I assume this was probably copy/pasted from the `cascade_destroy` module?

Reply:
Yep, sounds correct to me 😅